src/cmlibs/widgets/handlers/interactionmanager.py

```diff
@@ -24,7 +24,10 @@ def register_handler(self, handler):
 
         if self._fallback_handler is None:
             self._fallback_handler = handler
-            self._active_handler = handler
+            self._activate_handler(handler)
+
+    def clear_active_handler(self):
+        self.unregister_handler(self._active_handler)
```

Comment:
These lines are duplicated.

Reply:
This has been removed. I didn't write this, so I guess something weird happened with the merge. Everything else looks correct though.